ngageoint/opensphere
# perf(cesium): Use flat coords for multi-geom translation

All geom types now read from flat coordinates using the given offset, stride, and ends. Scratch coordinate arrays are used to prevent array creation during geometry translation. The greatest performance enhancement comes from Multi geoms, which previously called `multi.getGeometries()` which re-inflates the sub geos from the flat coordinates. Since we now use the flat coords directly, it is both faster and more memory-efficient.

I'm not overly happy with the fix, largely because `FeatureConverter` could use a design refactor in the first place.

Resolves #135

## Testing
* For every geometry type (including `MultiPoint`, `MultiLineString`, and `MultiPolygon`)
    * Ensure it is rendered identically to master
    * Ensure it can be picked/highlighted
    * Bring up the timeline and ensure it is rendered correctly and can be picked/highlighted
* Test KML track animation
* Test Hide/Show feature both in and out of the timeline
